src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -8,17 +8,36 @@ import { fullNamePipe } from '../../../utils/pipes';
 import PageBlock from '../../../layouts/PageBlock';
 import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material';
 import ReactHookTextField from '../../../components/ReactHookTextField';
-import { Control, Controller, FieldErrorsImpl } from 'react-hook-form';
+import { Controller } from 'react-hook-form';
 import { DatePicker } from '@mui/x-date-pickers';
-import { WorkPackageEditFormPayload } from './WorkPackageEditContainer';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 
 interface Props {
-  users: User[];
-  control: Control<WorkPackageEditFormPayload>;
-  errors: Partial<FieldErrorsImpl<WorkPackageEditFormPayload>>;
+  lead?: string;
+  manager?: string;
+  setManager: (manager?: string) => void;
+  setLead: (lead?: string) => void;
+  usersForProjectLead: User[];
+  usersForProjectManager: User[];
+  control: any;
```

Comment:
can you change these back to the types we added, last change and then I'll merge

Reply:
control and errors